packages/amazonq/src/extensionShared.ts

```diff
@@ -103,7 +103,10 @@ export async function activateShared(context: vscode.ExtensionContext, isWeb: bo
     await activateQGumby(extContext as ExtContext)
 
     // Generic extension commands
-    registerCommands(context, contextPrefix)
+    registerGenericCommands(context, contextPrefix)
+
+    // Amazon Q specific commands
+    registerCommands(context)
```

Comment:
@hayemaxi, I see you are removing `registerCommands` from the imports in line number 30 and adding registerGenericCommands. Is this expected? cc: @justinmk3

Reply:
Addressed in https://github.com/aws/aws-toolkit-vscode/pull/5080

Looks like we have identified a gap... our CI should be catching this. Nice catch, thanks!